requirements-setup.txt

```diff
@@ -1,13 +1,15 @@
 black==19.3b0
 click==6.*
 mypy==0.630
+notedown
```

Comment:
Is this part of this commit?

Reply:
No, this this is in a separate commit but I included it here as I didn't want to have a one-liner PR.